penthouse.gemspec

```diff
@@ -29,8 +29,11 @@ Gem::Specification.new do |spec|
   # web
   spec.add_development_dependency 'rack', '~> 1.6.4'
   # db
-  spec.add_development_dependency 'ar-octopus', '~> 0.8.6'
-  spec.add_development_dependency 'activesupport', '~> 4.2.6'
-  spec.add_development_dependency 'activerecord', '~> 4.2.6'
-  spec.add_development_dependency 'pg'
+  spec.add_dependency 'ar-octopus', '~> 0.8.6'
```

Comment:
Is there a reason this is a full dependency now? Not everyone will be using Octopus (i.e. those who just need Postgres Schemas)